CHANGELOG.asciidoc

```diff
@@ -25,6 +25,9 @@ image::https://raw.githubusercontent.com/apache/tinkerpop/master/docs/static/ima
 
 * Prevented XML External Entity (XXE) style attacks via `GraphMLReader` by disabling DTD and external entities by default.
 
+==== Bugs
```

Comment:
sorry - maybe i wasn't clear. we generate the "JIRA Section" of the CHANGELOG on release. you just need to write a friendly one liner bullet in the section just above this (i.e. above/below the "XML External Entity" entry.

Reply:
Ah ok, thanks. I updated it to be a simple sentence under the XML entry.